Docs: Plugins Page Filter Options Updates

dericksozo opened the pull request:
(docs): Updating the necssary screenshots, text, and titles to take into account the new Plugins category filter component. Making things more clear.

This PR takes care of Issue #425.

PazYanoverr commented on docs/getting-started/assets/installed-plugins.png:
@dericksozo - It's weird that the installed plugins contains also plugins which are not installed (see image)

dericksozo replied:
Can we get a developer's opinion on this? From what I saw in the app, the plugin remains installed but toggling it off puts it into a _disabled_ state.

PazYanoverr replied:
@dericksozo - You are right. The plugin remains installed even after toggling it off (@mulygottlieb - Do you know why? Do we still generate the code of "off" plugin?).
However, I think that in the documentation screenshot we should better avoid displaying this edge / confusing case

PazYanoverr commented on docs/getting-started/plugins.md:
```diff
@@ -12,46 +12,48 @@ Amplication uses _Plugins_ to extend the functionality of your generated code.
 Some plugins can be enabled when you first create your service.
 For example, if you choose MySQL as your database, the _MySQL DB_ plugin will be enabled.
-If you choose to turn on authentication for your service, that will enable the _NestJS Auth Module_ plugin.
+If you choose to turn on authentication for your service, that will enable the _NestJS Auth Module_ plugin and other Authentication-focused plugins.
```
```suggestion
If you choose to turn on authentication for your service, that will enable the _NestJS Auth Module_ plugin and other required Authentication-focused plugins.
```

PazYanoverr commented on docs/getting-started/plugins.md:
```diff
@@ -12,46 +12,48 @@ Amplication uses _Plugins_ to extend the functionality of your generated code.
-Your installed plugins are shown on the _Installed Plugins_ page.
-Plugins can be manually enabled or disabled on either of these pages.
+All community plugins published by Amplication are available on your service's _Plugins_ tab.
+Plugins can be enabled or disabled on this tab.
```
```suggestion
Plugins can be enabled or disabled on this tab. In addition- the settings of each plugin can be configured there
```

PazYanoverr commented on docs/getting-started/plugins.md:
```diff
@@ -12,46 +12,48 @@ Amplication uses _Plugins_ to extend the functionality of your generated code.
-Plugins can be manually enabled or disabled on either of these pages.
+All community plugins published by Amplication are available on your service's _Plugins_ tab.
+Plugins can be enabled or disabled on this tab.
+To view your installed plugins, click on the _Installed Plugins_ category in the filter options on the left side.
```
I'm not sure the "installed plugins" in a Category- as you can see the title of Categories **below** the "installed plugins" option. Maybe we should keep the "page" terminology for the "installed plugins" option. WDYT?
Note that this comment is relevant in other places of this PR as well

dericksozo commented:
Thank you @PazYanoverr. I've implemented your feedback. I also changed the terminology from `filter options` to `filter panel` so the word `page` makes more sense in this context.

dericksozo commented:
Closing this outdated PR as approved by Paz. Documentation has been updated elsewhere.
